packages/KInlineEdit/KInlineEdit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    id="editable-wrapper"
+    class="k-inline-edit">
+    <input
+      v-if="isEditing"
+      ref="input"
+      :style="styles"
+      v-model="inputText"
+      class="k-input"
+      @blur="handleSave"
```

Comment:
When I remove the text in the vuepress example, it emits an empty value '', but the kinput is reset to what it was before.
![Jun-15-2020 15-11-21](https://user-images.githubusercontent.com/5770711/84696451-885c9200-af1a-11ea-8cd9-0d97db60f78c.gif)

Reply:
hmm good find, thanks

Reply:
fixed this, wrong logic in the example